sympy/plotting/plot.py

```diff
@@ -1730,6 +1730,26 @@ def plot3d_parametric_surface(*args, **kwargs):
     return plots
 
 
+def conplot(*args, **kwargs):
+    """
+    Draws contour plot of a given function.
+    """
+    args = list(map(sympify, args))
+    symbols = set()
+    show = kwargs.pop('show', True)
+    plot_expr = check_arguments(args, 1, 2)
+    symbols |= plot_expr[0].free_symbols
```

Comment:
Yes , some part of the code wasn't required. I've modified it. 
All symbols that are part of the plot expression are independent variables so I don't see why they need to be in order.The code in `def plot3d` does not require it to be so. Or maybe I've misunderstood what you mean by order.